### PR TITLE
DROTH-1357 && DROTH-1356 Multiple selection toll, split and separete do not work properly

### PR DESCRIPTION
**Pull request by MAC-CGI**
DROTH-1357 && DROTH-1356 Multiple selection toll, split and separete do not work properly

### CODE REVIEW

**Comment by oskarmalmstrom** [comment]
ok for me

**Comment by marquesfm** [comment]
Ok for me

File: UI/src/model/selectedLinearAsset.js [hunk 1]
```diff
@@ -195,8 +196,10 @@
     this.cancel = function() {
       if (self.isSplit() || self.isSeparated()) {
         cancelCreation();
+        self.close();
       } else {
         cancelExisting();
+        self.close();
```
Comment (joanacoelho) [inline UI/src/model/selectedLinearAsset.js]: The self.close can be put in the end of the function: 
```
this.cancel = function() {
     if (self.isSplit() || self.isSeparated()) {
         cancelCreation();
      } else {
         cancelExisting();
       }
       self.close(); 
    }
```

File: UI/src/model/selectedLinearAsset.js [hunk 2]
```diff
@@ -323,6 +332,10 @@
       self.setValue(undefined);
     };
 
+    this.removeMultiValue = function() {
+      self.setMultiValue(undefined);
```
Comment (joanacoelho) [inline UI/src/model/selectedLinearAsset.js]: You can just call `self.setMultiValue();`

File: UI/src/application.js [hunk 3]
```diff
@@ -227,9 +227,7 @@
         zoom: startupParameters.zoom,
         resolutions: [2048, 1024, 512, 256, 128, 64, 32, 16, 8, 4, 2, 1, 0.5, 0.25, 0.125, 0.0625]
       }),
-      interactions: ol.interaction.defaults({
-        dragPan: false
-      })
+      interactions: ol.interaction.defaults({})
```
Comment (leirasf) [inline UI/src/application.js]: You can delete this line.

**Pull request merged**